.prettierignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
Is `node_modules` automagically ignored by Prettier? That'd be sikk ngl

Reply:
Yep, automagically ignored